### DISCUSSION TITLE
Crashing issues when using certain vehicles?

### DISCUSSION THREAD

**Discussion by Chibibibo** [Berny23/LD-ToyPad-Emulator#159] [Q&A]
Hi, I have issues with the game crashing in Cemu with certain vehicles and sometimes it just crashes randomly, I followed everything closely and even dumped my own copy of the game, but I keep running into crashes. Also, it asks me to connect to the internet for an update but I already have all the update data downloaded from my WiiU, is there something I can do to fix all this?

**Comment by Euniemeansme** [comment]
This is the Github for only the toypad emulator. If you're having issues with Cemu or games running on Cemu, please ask in the Cemu discord or their github. Our job is to help you get the toypad emulator working, not Cemu.

**Reply by Chibibibo** [reply]
Okay, I just assumed the issue was related to the toypad since I dont have this issue in other games, thanks for responding tho!
